components/CopyCodeBlock/copyCodeBlock.module.css

```diff
@@ -0,0 +1,51 @@
+/* Styles for the button before being clicked */
+
+.codeCopyBtn {
+	color: white;
```

Comment:
White text on a grey background is not readable. Please use an appropriate variable from the global css file